src/libs/OptionsListUtils.ts

```diff
@@ -515,10 +515,10 @@ function getAllReportErrors(report: OnyxEntry<Report>, reportActions: OnyxEntry<
             reportActionErrors.smartscan = ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage');
         }
     } else if ((ReportUtils.isIOUReport(report) || ReportUtils.isExpenseReport(report)) && report?.ownerAccountID === currentUserAccountID) {
-        if (ReportUtils.hasMissingSmartscanFields(report?.reportID ?? '') && !ReportUtils.isSettled(report?.reportID)) {
+        if (ReportUtils.shouldShowRBRForMissingSmartscanFields(report?.reportID ?? '') && !ReportUtils.isSettled(report?.reportID)) {
             reportActionErrors.smartscan = ErrorUtils.getMicroSecondOnyxError('report.genericSmartscanFailureMessage');
         }
-    } else if (ReportUtils.hasSmartscanError(Object.values(reportActions ?? {}))) {
+    } else if (ReportUtils.hasSmartscanError(Object.values(reportActions ?? {}), true)) {
```

Comment:
Are we calling `ReportUtils.hasSmartscanError` from anywhere else? Do we need the second param?

Even if we need it, I don't think it should be a separate param, and instead we should have that check outside of the `ReportUtils.hasSmartscanError` function, since that function name sounds like it shouldn't care about `isLHNPreview`

Reply:
> I think we still need to create a `isLHNPreview` param then handle both case (`isLHNPreview` is `true` or `false`) in `hasSmartscanError` to reduce the duplicate logics.

I think I'm not following or I'm missing something, because if we always pass this as true, this doesn't make sense